numpy-stubs/__init__.pyi

```diff
@@ -497,6 +500,13 @@ def ones_like(
     subok: bool = ...,
     shape: Optional[_ShapeLike] = ...,
 ) -> ndarray[int]: ...
+def empty_like(
+    a: ArrayLike,
+    dtype: DtypeLike = ...,
+    order: str = ...,
+    subok: bool = ...,
+    shape: Optional[_ShapeLike] = ...,
+) -> ndarray[int]: ...
```

Comment:
Hm the `ndarray[int]` in `ones_like` is actually a mistake (we haven't made `ndarray` generic over dtype yet). Mind removing the `[int]` part from `empty_like` and (while we're here) `ones_like`?

Reply:
Oh, sorry. I'll fix both.